mpkey.txt

```diff
@@ -0,0 +1 @@
+7c1cbc95d6519b7d6417d6fe1ce393e3
```

Comment:
Might be more clean for this to be left out and only built into the production version of Exokit?

Reply:
This can be a JSON.

Reply:
> Might be more clean for this to be left out and only built into the production version of Exokit?

Yeah, that's probably the route to take. For dev + forks we don't want to be doing analytics.

> This can be a JSON.

Indeed 👍.